### PR TITLE
[READY-TO-MERGE] Document templateInfo in Space type

### PR DESCRIPTION
**Pull request by a-kent**
@andrewegan Can you do an initial review?

Was in two minds as to the most appropriate place to document - I see, for example, that the `Message` guide includes some `experimental` fields in the main (`published`) doc - but I've made this a guide in its own right at least for starters: (a) so as not to pollute the main `Space` documentation, and (b) because there's a dedicated `SpaceTemplateInfo` type to be documented here.

What do you think?

### CODE REVIEW

File: guides/V1_space_template_info.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Example request
+
+~~~~
+Method: POST
+URL: https://api.watsonwork.ibm.com/graphql
+Headers: 'Content-Type: application/graphql' , 'x-graphql-view: PUBLIC, EXPERIMENTAL'
+Body:
+{
+  space(id: "space-id") {
```
Comment (andrewegan) [inline guides/V1_space_template_info.md]: I already think this is getting very costly to maintain with the same data copy-pasted across files. 

I think we should look at some template engine if we need to have this information on all pages. For now I suggest we just link to the explorer main page.

```graphql
type Space {
...
  templateInfo: SpaceTemplateInfo
}
```
I think this should be enough for anyone somewhat familiar with GraphQL.

File: guides/V1_space_template_info.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-info'
+is: 'experimental'
+---
+
+# Space Template Info
+
+In addition to those space properties documented publicly, you can add _templateInfo_ to your _space_ request in order to fetch
+information about the template used to create a space.
```
Comment (andrewegan) [inline guides/V1_space_template_info.md]: > those space properties documented publicly

It's not clear what this refers to.

----

> information about the template used to create a space.

How about 

> [...] the template associated with the space

File: guides/V1_space_template_info.md (hunk 1 above)
Comment (andrewegan) [reply guides/V1_space_template_info.md]: We could have a generic example for posting a query instead of one for each query.

Comment (a-kent) [reply guides/V1_space_template_info.md]: Removed the example request and replaced with a link to explorer main page.

File: guides/V1_space_template_info.md (hunk 2 above)
Comment (a-kent) [reply guides/V1_space_template_info.md]: Amended. (Was referring to properties in the "published" Space guide, but removed that piece.)

**Comment by a-kent** [comment]
@jarusso @miguelestrada Can you review please?

**Review by miguelestrada — APPROVED** [review]
The info is accurate and allows us to move forward, but we do need to augment this with a higher context set of information with regards to the role of templates and spaces, when to use them and why, use cases that drove this, etc..

**Pull request merged**